impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -70,3 +70,15 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domain/info" };
   }
 }
+
+// The KeyTransparencyAdminService API provides batch access to the directory of public keys. 
+service KeyTransparencyAdminService {
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple entries at once.  
+  rpc BatchSetEntries(keytransparency.v1.types.BatchSetEntriesRequest) returns (keytransparency.v1.types.BatchSetEntriesResponse) {
```

Comment:
How do you feel about calling this `BatchUpdateEntries`. Set gives the impression that this API should be called once at the beginning, but update hints that it should be called every time there's a entry update.